src/main/holochainManager.ts

```diff
@@ -374,14 +390,23 @@ export class HolochainManager {
     });
   }
 
-  async installWebhappFromHashes(
-    happSha256: string,
-    uiZipSha256: string,
-    appId: string,
-    distributionInfo: DistributionInfoV1,
-    networkSeed?: string,
-    membrane_proofs?: { [key: string]: MembraneProof },
-  ): Promise<void> {
+  async installWebhappFromHashes({
```

Comment:
Out of curiosity, what is the reason for changing the input to an object?

Reply:
ok so it was because we would need to call it 

(happSha256, uiZipSha256, appId, distributionInfo, networkSeed, undefined, icon), when we have object, we can do just ({happSha256, uiZipSha256, appId, distributionInfo, networkSeed, icon})